src/ZoneServer/Network/Send.cs

```diff
@@ -3180,20 +3220,34 @@ public static void ZC_TRUST_INFO(IZoneConnection conn)
 		/// <param name="animationName">Name of the animation to play (uses packet string database to retrieve the id of the string).</param>
 		/// <param name="stopOnLastFrame">If true, the animation plays once and then stops on the last frame.</param>
 		public static void ZC_PLAY_ANI(IActor actor, string animationName, bool stopOnLastFrame = false)
+		{
+			if (!ZoneServer.Instance.Data.PacketStringDb.TryFind(animationName, out var packetStringData))
+				throw new ArgumentException($"Unknown packet string '{animationName}'.");
+
+			ZC_PLAY_ANI(actor, packetStringData.Id, stopOnLastFrame);
+		}
+
+		/// <summary>
+		/// Plays animation for actor on nearby clients.
+		/// </summary>
+		/// <param name="actor">Entity to animate.</param>
+		/// <param name="packetStringId">Id of the string for the animation to play.</param>
+		/// <param name="stopOnLastFrame">If true, the animation plays once and then stops on the last frame.</param>
+		/// <param name="unknowBoolean">Unknow purpose at this moment.</param>
+		public static void ZC_PLAY_ANI(IActor actor, int packetStringId, bool stopOnLastFrame = false, bool unknowBoolean = false)
 		{
 			var packet = new Packet(Op.ZC_PLAY_ANI);
 
 			packet.PutInt(actor.Handle);
-			packet.AddStringId(animationName);
+			packet.PutInt(packetStringId);
 			packet.PutByte(stopOnLastFrame);
 			packet.PutByte(0);
 			packet.PutFloat(0);
 			packet.PutFloat(1);
 
-			// [i373230] Maybe added earlier
+			// [i373230] Maybe added earlier [Updated ib 28/07/24]
 			{
-				packet.PutByte(0);
-				packet.PutByte(0);
+				packet.PutShort(unknowBoolean ? 248 : 0);
```

Comment:
Making a short makes the life easier since I could directly sign it to 248 if needed.
But it's part of the process, trial and error. I can't magically identify what's it's purpose so I left as a unknow.

Reply:
> I can't magically identify what's it's purpose

And yet you found the need to change this, so I must assume that you did "magically" figure out _something_^^ The alternative is that you changed this for no reason. Either way, it doesn't make sense, and it's unrelated to anyone understanding what these two bytes do in actuality. You didn't tell us why this was changed, what the change was supposed to accomplish, and as I described, you opened up a lot of potential questions, which is the real problem.